src/hotspot/share/logging/logConfiguration.cpp

```diff
@@ -636,10 +636,15 @@ void LogConfiguration::print_command_line_help(outputStream* out) {
   out->cr();
 
   out->print_cr("Asynchronous logging (off by default):");
-  out->print_cr(" -Xlog:async");
+  out->print_cr(" -Xlog:async[:[mode]]");
```

Comment:
This reminded me we need an update to the Java manpage/command-reference as well.

Reply:
Fixed! See picture for a render.

![image](https://github.com/user-attachments/assets/105ff56a-d79e-49ee-bc91-89733af5b1b0)

Reply:
That's `man` opening the generated man page file from markdown.